account_reconcile_oca/views/account_bank_statement_line.xml

```diff
@@ -19,20 +19,16 @@
                 <field name="aggregate_name" />
                 <templates>
                     <t t-name="kanban-box">
-                        <div
```

Comment:
personnaly I found the use of the div more visible on you scree.

Reply:
@flotho ,sorry didn't get you. Can you please elaborate a bit?